### PR TITLE
RAC-381: Use DSM Modal in Measurements

### PR DESCRIPTION
**Pull request by ValentinMumble**
**Description (for Contributor and Core Developer)**

🔥

- Removing @shared Modal and using DSM's one
- Add required title prop on IconButton and in Modal

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

**Review by juliensnz — APPROVED** [review]

File: src/Akeneo/Tool/Bundle/MeasureBundle/Resources/public/shared/components/TextField.tsx [hunk 1]
```diff
@@ -5,6 +5,12 @@ import {LockIcon} from 'akeneo-design-system';
 import {useTranslate} from '@akeneo-pim-community/legacy-bridge';
 import {ValidationError, InputErrors} from '@akeneo-pim-community/shared';
 
+const Container = styled.div`
```
Comment (StevenVAIDIE) [inline src/Akeneo/Tool/Bundle/MeasureBundle/Resources/public/shared/components/TextField.tsx:8]: Sure about it ? AknFieldContainer have a lot of other CSS

File: src/Oro/Bundle/PimDataGridBundle/Resources/public/js/datagrid/quickexport/component/QuickExportConfigurator.tsx [hunk 2]
```diff
@@ -77,78 +29,74 @@ const QuickExportConfigurator = ({
...
+              onActionLaunch(formValue);
+              closeModal();
+            }}
+            disabled={!readyToSubmit}
+          >
+            {translate('pim_common.export')}
+          </Button>
+        </Modal.TopRightButtons>
+        <SectionTitle color="brand">
+          {translate('pim_datagrid.mass_action.quick_export.configurator.subtitle')}&nbsp;|&nbsp;
```
Comment (StevenVAIDIE) [inline src/Oro/Bundle/PimDataGridBundle/Resources/public/js/datagrid/quickexport/component/QuickExportConfigurator.tsx:53]: ```&nbsp;|&nbsp;``` :eyes: :smile:

**Review by StevenVAIDIE — APPROVED** [review]

File: src/Akeneo/Tool/Bundle/MeasureBundle/Resources/public/shared/components/TextField.tsx (hunk 1 above)
Comment (ValentinMumble) [reply src/Akeneo/Tool/Bundle/MeasureBundle/Resources/public/shared/components/TextField.tsx:8]: Yes, I checked, this component is only used in Measurements and the appearance did not change so let's gooooo
They are also destined to be trashed eventually 🔥

**Review by tseho — APPROVED** [review]

File: src/Oro/Bundle/PimDataGridBundle/Resources/public/js/datagrid/quickexport/component/QuickExportConfigurator.tsx (hunk 2 above)
Comment (ValentinMumble) [reply src/Oro/Bundle/PimDataGridBundle/Resources/public/js/datagrid/quickexport/component/QuickExportConfigurator.tsx:53]: It was either this or the loooong template string that was before `{`${translate('pim_datagrid.mass_action.quick_export.configurator.subtitle')} | ${translate('pim_common.result_count',   ...  )}`}`
I find it easier to read 🤔

**Pull request merged**